### ISSUE TITLE
tasks-app

### ISSUE THREAD

**Issue by ausbuscon** [sonrisesoftware/tasks-app#53]
Just installed into Ubuntu Desktop - it installs OK but does not execute??

Ubuntu-Touch device it runs fine

**Comment by iBelieve** [comment]
What Ubuntu version are you using?

**Comment by ausbuscon** [comment]
14.04

**Comment by iBelieve** [comment]
Did you install it from the Ubuntu Software Center or from GitHub?

**Comment by ausbuscon** [comment]
From the software centre

Thanks for your help with this.

**Comment by iBelieve** [comment]
I'm really sorry for the problem. It turns out it is a bug in the packaging Canonical did for Tasks. I'll get an update out but it may be several weeks or a month before it gets reviewed at the rate that Canonical is currently reviewing apps and updates.

So, if you want, here is how you can fix it:

Open a Terminal and type: `sudo nano /opt/ubuntu-tasks/launcher.sh`

Then you should see a line that says:

```
/usr/bin/qmlscene -qt=$(qtchooser -list-versions | grep linux) run /opt/ubuntu-tasks/ubuntu-tasks.qml
```

Replace the part with the dollar symbol and in parentheses to match this line:

```
/usr/bin/qmlscene -qt=qt5 run /opt/ubuntu-tasks/ubuntu-tasks.qml
```

Then type `Ctrl+O` and then `Ctrl+X` to save your changes and exit the program.

Once again, I'm sorry this happened and I'll get an update out right away.

**Comment by ausbuscon** [comment]
worked well - thanks again

**Comment by ausbuscon** [comment]
Hi Michael,

Pardon my ignorance - but how does the syncing work between my touch device
and my ubuntu desktop

ta
Steven

**Comment by iBelieve** [comment]
So sorry for the confusion on this. When I first built the app, there wasn't any good documentation on how the syncing was supposed to work, so I had assumed it was automatic. It turns out that it is not automatic, and I as the developer will need to had extra code to make the syncing work.

I have asked the Software Center moderators to remove the part of the description about syncing from the app's description, but it hasn't happened yet because they are very busy on other projects.

I'm currently working on other projects, but I will try to work on the sync for the tasks app sometime soon.

**Comment by ausbuscon** [comment]
Hi Michael

thanks for the info -  are you working on email (hope so - really looking
forward to that)!!

BTW, I seem to have lost the settings swipe in the ubuntu-touch version of
tasks (i.e. don't have the option to connect to trello)

Appreciate the work you have done with this app

Steven

**Comment by iBelieve** [comment]
What do you mean that I am working on email? Are you asking if I am working on the Email client for Ubuntu?

Do you mean how you swipe up from the bottom of the screen to get to the gray overlay with different actions? That was removed by Ubuntu, so I will need to find a work around

**Comment by ausbuscon** [comment]
Hi Michael,

I'm impressed with tasks, and I was wondering if your ingenuity was being
applied to the email client.

In relation to the second paragraph, yes that's what I mean. With a fresh
install, tasks wil no longer connect to Trello.
I'll put something on the review area so ubuntu know as well.

Cheers
Steven

**Comment by iBelieve** [comment]
No, I have not been working on the email client. I have way too many projects as it is :)
